lib/square_connect/api_client.rb

```diff
@@ -27,12 +24,13 @@ class ApiClient
     # @return [Hash]
     attr_accessor :default_headers
 
+    # Initializes the ApiClient
+    # @option config [Configuration] Configuration for initializing the object, default to Configuration.default
     def initialize(config = Configuration.default)
       @config = config
-      @user_agent = "square-connect-ruby/#{VERSION}"
```

Comment:
Let's make sure that this change is not something we made intentionally in our original fork of the SDK templates. IIRC we made some change around user agents.